src/models/incident/saga.js

```diff
@@ -49,6 +49,13 @@ export function* patchIncident(action) {
     yield put(patchIncidentSuccess({ type: payload.type, incident }));
     yield put(requestHistoryList(payload.id));
   } catch (error) {
+    const { response } = error;
+
+    if (response.status === 403) {
+      const { jsonBody } = response;
+      global.alert(jsonBody.detail);
```

Comment:
Would leave this out

Reply:
Why, exactly? Showing an alert window has been discussed with UX and has been approved.